t/17_apps/02_load_app.t

```diff
@@ -29,8 +29,8 @@ ok defined($forum), "app 'Forum' is defined";
 is @{ $main->registry->routes->{'get'} }, 1, 
     "one route is defined in main app";
 
-is @{ $test_app->registry->routes->{'get'} }, 16, 
-    "16 routes are defined in test app";
+is @{ $test_app->registry->routes->{'get'} }, 18, 
+    "18 routes are defined in test app";
```

Comment:
Trivial, but Test::More::is() will tell you that it should have been 18 but isn't, so you don't need to mention that in the test title as well.